### PR TITLE
use new BootOptions

### PR DESCRIPTION
**Pull request by victor-stone**
Not 100% sure if I did this right but it works for the latest canary

### CODE REVIEW

File: app/initializers/fastboot.js [hunk 1]
```diff
@@ -40,7 +40,10 @@ export default {
 
         var promise;
         Ember.run(function() {
-          promise = App.visit(url);
+          promise = App.visit(url {
```
Comment (topaxi) [inline app/initializers/fastboot.js]: Missing comma :)

**Comment by rwjblue** [comment]
@chancancode - r?

**Comment by chancancode** [comment]
I'm working on a more complete patch: https://github.com/tildeio/ember-cli-fastboot/pull/71

I'll be on a plane soon, so I pushed it up so you can start poking at it. (It might or might not work, but that's the broad strokes)

**Comment by rwjblue** [comment]
Thanks @chancancode!

@victor-stone - I'm going to close this in favor of #71.  We can always revisit (pun intended) if #71 ends up taking too long...

**Pull request closed without merging**